### ISSUE TITLE
The composite video is missing characters

### ISSUE THREAD

**Issue by Rookie-Kai** [Fictionarry/ER-NeRF#17]
Hello, thank you very much for your work, but I had some problems when synthesizing the video. I used Obama as the video content and reasoned through my own audio. After Obama video processing according to your preprocessing method and obtaining AU.csv file, I reasoned using ngp.pth in trial_obama_torso and ngp_ep0028.pth, and the resulting videos were all missing people, only backgrounds.

Is there something wrong with my operation? How should I handle this?

https://github.com/Fictionarry/ER-NeRF/assets/43467822/f3d9edc7-c983-47f1-941b-38ab076b8f0b

This is my log information：

> Namespace(path='data/obama/', O=True, test=True, test_train=True, data_range=[0, -1], workspace='trial_obama_torso/', seed=0, iters=200000, lr=0.01, lr_net=0.001, ckpt='latest', num_rays=65536, cuda_ray=True, max_steps=16, num_steps=16, upsample_steps=0, update_extra_interval=16, max_ray_batch=4096, warmup_step=10000, amb_aud_loss=1, amb_eye_loss=1, unc_loss=1, lambda_amb=0.0001, fp16=True, bg_img='', fbg=False, exp_eye=True, fix_eye=-1, smooth_eye=False, torso_shrink=0.8, color_space='srgb', preload=0, bound=1, scale=4, offset=[0, 0, 0], dt_gamma=0.00390625, min_near=0.05, density_thresh=10, density_thresh_torso=0.01, patch_size=1, init_lips=False, finetune_lips=False, smooth_lips=False, torso=True, head_ckpt='', gui=False, W=450, H=450, radius=3.35, fovy=21.24, max_spp=1, att=2, aud='data/1.npy', emb=False, ind_dim=4, ind_num=10000, ind_dim_torso=8, amb_dim=2, part=False, part2=False, train_camera=False, smooth_path=False, smooth_path_window=7, asr=False, asr_wav='', asr_play=False, asr_model='deepspeech', asr_save_feats=False, fps=50, l=10, m=50, r=10)
Setting up [LPIPS] perceptual loss: trunk [alex], v[0.1], spatial [off]
/HOME/scz0bse/.conda/envs/ernerf/lib/python3.10/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and will be removed in 0.15, please use 'weights' instead.
  warnings.warn(
/HOME/scz0bse/.conda/envs/ernerf/lib/python3.10/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and will be removed in 0.15. The current behavior is equivalent to passing `weights=AlexNet_Weights.IMAGENET1K_V1`. You can also use `weights=AlexNet_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Loading model from: /HOME/scz0bse/.conda/envs/ernerf/lib/python3.10/site-packages/lpips/weights/v0.1/alex.pth
Setting up [LPIPS] perceptual loss: trunk [alex], v[0.1], spatial [off]
/HOME/scz0bse/.conda/envs/ernerf/lib/python3.10/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and will be removed in 0.15, please use 'weights' instead.
  warnings.warn(
/HOME/scz0bse/.conda/envs/ernerf/lib/python3.10/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and will be removed in 0.15. The current behavior is equivalent to passing `weights=AlexNet_Weights.IMAGENET1K_V1`. You can also use `weights=AlexNet_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Loading model from: /HOME/scz0bse/.conda/envs/ernerf/lib/python3.10/site-packages/lpips/weights/v0.1/alex.pth
[INFO] Trainer: ngp | 2023-08-09_10-21-41 | cuda | fp16 | trial_obama_torso/
[INFO] #parameters: 1787681
[INFO] Loading latest checkpoint ...
[WARN] No checkpoint found, model randomly initialized.
[INFO] load 7272 train frames.
[INFO] load data/1.npy aud_features: torch.Size([121, 29, 16])
...
Loading train data: 100%|██████████| 7272/7272 [04:19<00:00, 28.00it/s]
[INFO] eye_area: 0.0 - 1.0
==> Start Test, save results to trial_obama_torso/results

100% 121/121 [00:44<00:00,  7.09it/s][swscaler @ 0x629cb00] Warning: data is not aligned! This can lead to a speed loss
[swscaler @ 0x59b8bc0] Warning: data is not aligned! This can lead to a speed loss
==> Finished Test.

100% 121/121 [00:48<00:00,  2.52it/s]

Thank you for taking time out of your busy schedule

**Comment by Fictionarry** [comment]
> [INFO] Loading latest checkpoint ...
[WARN] No checkpoint found, model randomly initialized.

Seems the program failed in loading the checkpoint file and so no contents are rendered. You need to put the.pth files into `./trial_xxx/checkpoints` and the program will find it.
